indigo/tyrian-sandbox/src/main/scala/example/IndigoSandbox.scala

```diff
@@ -182,8 +231,12 @@ final case class Model(
     bridge: TyrianIndigoBridge[IO, String, Unit],
     field: String,
     components: List[Counter.Model],
-    randomNumber: Int
+    randomNumber: Int,
+    ensureGame1Div: Boolean,
+    ensureGame2Div: Boolean,
+    game1: Option[MyAwesomeGame],
+    game2: Option[MyAwesomeGame]
```

Comment:
This is probably _not_ the most elegant modelling that could be achieved here. 😁